packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -73,7 +73,14 @@ export const Alert = React.forwardRef<HTMLDivElement, Props>(
         {/* If onRemove is specified, giving preference to onRemove */}
         {onRemove && !buttonContent && (
           <div className={styles.close}>
-            <IconButton aria-label="Close alert" name="times" onClick={onRemove} size="lg" type="button" />
+            <Button
```

Comment:
What was the reason for changing to `Button`? Is it just the "squareness" of it? Just curious for our own investigation of unifying these two options

Reply:
@JoaoSilvaGrafana yea, two reasons the squareness (which looks good when it matches the shape of the corner it's in) and the color of the hover effect for a secondary fill button is transparent so it takes on the color tint of the background which looks really good.